CHANGELOG.asciidoc

```diff
@@ -19,6 +19,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 ==== Added
 - Support for uploading sourcemaps {pull}302[302].
 - app.environment {pull}366[366]
+- Support for uploading `transaction.marks` {pull}430[430].
```

Comment:
I suggest: "Added support for `transaction.marks`"

Reply:
This is already in the section `Added` so I don't see value in repeating this, but will remove the `uploading`.